.github/workflows/hf.yml

```
@@ -0,0 +1,42 @@
+#name: Push to huggingface
```

actionlint (1.7.4)

workflow is empty

(syntax-check)